class/capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.upcase
+  end
```

Comment:
- [ ] Ouch! This is a close one, but there's a slight adjustment needed in your implementation. The method you used, @nameable.correct_name.upcase, converts the entire string to uppercase. However, based on the requirement, we should capitalize the output of @nameable.correct_name instead. 
- [ ] To achieve this, you should use the `capitalize` method instead of `upcase`. The capitalize method will capitalize the first letter of the string while keeping the rest of the characters in lowercase.